### DISCUSSION TITLE
The new app enumeration approach happens to make windows easier to implement

### DISCUSSION THREAD

**Discussion by riotrah** [will-stone/browserosaurus#603] [General]
Hey @will-stone,

Since I can't/don't know how to reply to individual commits or release descriptions, etc, wasn't sure where best to post this sort of musing or non-actionable thought, but the new approach to finding which installed apps the user has is pretty much what I do in Windows, but it searches through a greater variety of install locations, because of course Windows doesn't have a single consistent location for binaries lol. Not sure what I intended on you doing with that but, uh, it's interesting I guess 🤷.

A follow-up musing, not really for you in particular either: I wonder if it might be valuable to create some kind of library that attempts to be cross-platform, that allows you to look up whether an app is installed with ID or binary name, and has a per-OS solution for enumerating through package managers, common application binary installation folders, or other means that might be pseudo reliable, for either my fork, or other apps that depend on enumerating install applications in js 🤔.

Either way, Hope you've been well! 

PS: And if you're interested, my work on my fork has stalled for a variety of personal reasons but also feeling somewhat stuck at how hard it is to figure out how to have an electron app reliably be responsible for responding to HTTP protocol invocations on Windows :/

**Comment by riotrah** [comment]
Sorry for the double post but it makes it easier to quote reply to this specific comment:

In the original closed issue #368 where we first discussed some steps I could take and some progress about my Windows version, you'd mentioned a desire to keep the the projects separate, but thought of a good way to offer a nod back to this with a reminiscent name. 

What do you now think about potentially creating a discussion post or issue or something, perhaps this very one if edited, such that people have a means by which to discover if there is any Windows efforts, wherein I could post my fork (and request advice or contributions or testers, and what have you, if just a simple link)? 

(With all caveats and such regarding unrelated ownership and support questions, whatever)

To clarify, there's nothing specific to this point in time that makes me want to ask that now any more so than before - I suppose aside from my having posted the OP of this discussion reminding me my fork exists.

**Reply by will-stone** [reply]
Oooo, and you have a logo! 😎 

Happy to add a link to Browseratops on the README: https://github.com/will-stone/browserosaurus#notable-forks

**Reply by riotrah** [reply]
Oh sweet thanks - tho forgive my bashfulness, my fork doesn't work on Windows properly 😓. Would hate to false advertise as a _working_ fork vs a _work in progress_ one. 

It runs, and works if one manually invokes the selector with a URL (eg when debugging), but I can't get Windows to properly recognize it as the default browser or even a candidate for one.

It does however seem to fully work on mac, but I haven't merged or pulled from yours in a hot minute, missing out on some of the really sweet aesthetics you've got going on as of late!

**Reply by will-stone** [reply]
Okay, I’ll update with a WIP tag 👍
